src/raqm.h

```diff
@@ -81,10 +98,14 @@ typedef enum
  */
 typedef struct raqm_glyph_t {
     unsigned int index;
+    int visual_index;
+    int line;
```

Comment:
Move those two members to the end of the struct and mark them private for GTK-Doc.